src/components/Filters/FilterRow/FilterRow.tsx

```diff
@@ -30,42 +32,114 @@ const SplitField = styled.div<SplitFieldProps>`
   }
 `;
 
+const getDefaultCondition = (dataPoints, dataPointValue) =>
+  dataPoints
```

Comment:
let's create memoized `getDatapointConditions` since it is used here and also in next `getConditionInput`
```js
const getDatapointConditions = R.memoizeWith((dataPoints, dataPointValue) => R.pipe(
  R.find(R.propEq('value', dataPointValue)),
  R.prop('conditions')
)(dataPoints)
```

Reply:
Perfect, I've been a bit worry with the code optimization, a lot of listings. There are also other adepts for Ramda, recommendations are appreciated